config/config-package.py

```diff
@@ -335,6 +345,24 @@ def gha_workflows(self):
         folder = self.path / '.github' / 'workflows'
         folder.mkdir(parents=True, exist_ok=True)
         destination = folder / 'meta.yml'
+        options = self._get_options_for(
+            'github',
+            (
+                'branch',
+                'jobs',
+                'additional_jobs',
+            )
+        )
+        options.update(self._test_cfg())
+        if not options.get("branch"):
+            options['branch'] = GHA_DEFAULT_BRANCH
```

Comment:
The `ref` suggestion:

```suggestion
        if not options.get("ref"):
            options['branch'] = GHA_DEFAULT_REF
```